.github/workflows/release.yml

```diff
@@ -1,10 +1,6 @@
 name: release
 
 on:
-  pull_request:
-    paths:
-    - 'scripts/release_testing_setup.sh'
-    - '.github/workflows/release.yml'
```

Comment:
Aren't these needed to test changes to the release workflow?

Reply:
Yes, that is for the case that multiple presubmits will interfere each other, but since the change is not very frequent, will revert the change.